governance_artifacts/pilot/run_pilot_acceptance_gates.py

```
@@ -0,0 +1,282 @@
+    tf = ROOT / "governance_blueprint" / "terraform"
+    rc, out = _run(["terraform", "init", "-backend=false", "-input=false", "-no-color"], cwd=tf)
+    if rc != 0:
+        return False, f"terraform init failed: {out.strip().splitlines()[-1] if out.strip() else rc}"
+    rc, out = _run(["terraform", "validate", "-no-color"], cwd=tf)
+    ok = rc == 0 and ("Success" in out or "valid" in out.lower())
+    return ok, out.strip().splitlines()[-1] if out.strip() else f"rc={rc}"
+
+
+def check_opa_gates() -> tuple[bool, str]:
+    rc, out = _run(["opa", "test", str(GA / "rego")])
+    line = next((l for l in out.splitlines() if l.startswith("PASS:") or l.startswith("FAIL")), out.strip()[-80:])
```

Ruff (0.15.20)

[error] Ambiguous variable name: `l`

(E741)